### DISCUSSION TITLE
Multiple multiselects?

### DISCUSSION THREAD

**Discussion by Pan-nav** [janosh/svelte-multiselect#260] [Q&A] [answered]
Hey, I am relatively new to web development and svelte. I was wondering if there's an alternative to variable selected for bind:selected. This is because I wanted to have multiple multiselect for my project, but seems like no other variable name works?

**Comment by Pan-nav** [comment]
ps: I am extremely sorry if I'm being ignorant to something in the docs about this, I tried searching and using different ways to test other variables.

**Comment by janosh** [comment, marked as answer]
Use `<Multiselect bind:selected={myArray} />` to assign to a different name in the parent scope.
